hieradata/role.ci-slave.3.yaml

```diff
@@ -1,5 +1,19 @@
 ---
 ci_environment::jenkins_slave::labels:
   - 'elasticsearch-1.4'
-  - 'mongodb-2.4'
+  - 'mongodb-3.2'
   - 'postgresql-9.3'
+
+apt::sources:
+  'mongodb3.2':
+    location: 'http://repo.mongodb.org/apt/ubuntu/'
```

Comment:
Yeah, I had to do some weirdness [here](https://github.com/alphagov/govuk-puppet/blob/master/modules/mongodb/manifests/repository.pp#L29-L35)